packages/image_picker/image_picker/lib/image_picker.dart

```diff
@@ -206,13 +210,17 @@ class ImagePicker {
     double? maxHeight,
     int? imageQuality,
     CameraDevice preferredCameraDevice = CameraDevice.rear,
+    bool requestFullMetadata = true,
   }) {
-    return platform.getImage(
+    return platform.getImageFromSource(
```

Comment:
You need to require the version of the platform interface that has this method in pubspec.yaml.